pkg/apis/hive/v1/aws/machinepool.go

```diff
@@ -16,6 +16,18 @@ type MachinePoolPlatform struct {
 
 	// EC2RootVolume defines the storage for ec2 instance.
 	EC2RootVolume `json:"rootVolume"`
+
+	// SpotMarketOptions allows users to configure instances to be run using AWS Spot instances.
+	*SpotMarketOptions `json:"spotMarketOptions,omitempty"`
```

Comment:
Please give an explicit name to this field.

Reply:
Done